docs/user/01-10-installation.md

```diff
@@ -0,0 +1,28 @@
+---
+Install BPT Operator
+---
+
+## Prerequisites
+
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- Kubernetes cluster, or [k3d](https://k3d.io) for local installation
+  
+To enable the BTP Operator module from the latest release, you must install BTP Manager and SAP BTP Service Operator. Follow these steps to do that:
+
+1. To install BTP Manager, use the following command:
+
+    ```bash
+    kubectl apply -f https://github.com/kyma-project/btp-manager/releases/latest/download/rendered.yaml
```

Comment:
in comparison to /contributor installation prerequisite.yaml is not present. Have u tested that on pure k3d cluster?

Reply:
I tested that on pure k3d, errors about missing kyma-system namespace:
```
customresourcedefinition.apiextensions.k8s.io/btpoperators.operator.kyma-project.io created
clusterrole.rbac.authorization.k8s.io/btp-manager-manager-role created
clusterrolebinding.rbac.authorization.k8s.io/btp-manager-manager-rolebinding created
Error from server (NotFound): error when creating "https://github.com/kyma-project/btp-manager/releases/latest/download/rendered.yaml": namespaces "kyma-system" not found
Error from server (NotFound): error when creating "https://github.com/kyma-project/btp-manager/releases/latest/download/rendered.yaml": namespaces "kyma-system" not found
Error from server (NotFound): error when creating "https://github.com/kyma-project/btp-manager/releases/latest/download/rendered.yaml": namespaces "kyma-system" not found
Error from server (NotFound): error when creating "https://github.com/kyma-project/btp-manager/releases/latest/download/rendered.yaml": namespaces "kyma-system" not found
Error from server (NotFound): error when creating "https://github.com/kyma-project/btp-manager/releases/latest/download/rendered.yaml": namespaces "kyma-system" not found
```

Reply:
I see keda-manager have the same issue